### ISSUE TITLE
Create missing locale file

### ISSUE THREAD

**Issue by last-partizan** [pixari/vue-i18n-extract#114]
I'm using this tool for per-component translation, like this:

```
vue-i18n-extract -a -v src/ComponentName.vue -l locale/en/ComponentName.yml
```

For new components i must create empty yml file, but it would be nice to add option for creating missing file.

**Comment by rlam3** [comment]
@last-partizan could you share your approach with a repo? I'm interested in this per component translation as well. 
thanks!

**Comment by pixari** [comment]
Would you like to implement this feature, @last-partizan and/or @rlam3?

**Comment by last-partizan** [comment]
@pixari yes, i'll take a look when i'll have time (probably on weekends).

**Comment by pixari** [comment]
Sounds amazing @last-partizan :)

**Comment by magistr4815** [comment]
@pixari Hello. It's been a long time since the last reply. Is there any progress?
